src/utils/DateSlotMetrics.js

```diff
@@ -4,7 +4,8 @@ import { eventSegments, endOfRange, eventLevels } from './eventLevels'
 
 let isSegmentInSlot = (seg, slot) => seg.left <= slot && seg.right >= slot
 
-const isEqual = (a, b) => a.range === b.range && a.events === b.events
+const isEqual = (a, b) =>
```

Comment:
This is the migration. Previously a/b were just props objects and now they are wrapped in an array